### DISCUSSION TITLE
[Bug]: Certificate client as all expired

### DISCUSSION THREAD

**Discussion by g-pane** [angristan/openvpn-install#1095] [Help]
### Make sure your check these beforehand!

- [X] Issues - https://github.com/angristan/openvpn-install/issues
- [X] README and FAQ - https://github.com/angristan/openvpn-install
- [X] Wiki - https://github.com/angristan/openvpn-install/wiki
- [X] Discussions - https://github.com/angristan/openvpn-install/discussions

### What is the bug?

A bug happened!
`
Tue Dec 27 08:49:55 2022 VERIFY ERROR: depth=0, error=certificate has expired: CN=server_jl0uHOHQA5KVDK9v
Tue Dec 27 08:49:55 2022 OpenSSL: error:1416F086:SSL routines:tls_process_server_certificate:certificate verify failed
Tue Dec 27 08:49:55 2022 TLS_ERROR: BIO read tls_read_plaintext error
Tue Dec 27 08:49:55 2022 TLS Error: TLS object -> incoming plaintext read error
Tue Dec 27 08:49:55 2022 TLS Error: TLS handshake failed
`

All client certificare as expired! I can't connect without all the clients
How can i solve?

Can i disable temporany the client authentication certificate?
Can i move the datetime to a previus date?
Can i renew without deploy all the 200 certificate?

### Relevant log output

```shell
Tue Dec 27 08:49:55 2022 VERIFY ERROR: depth=0, error=certificate has expired: CN=server_jl0uHOHQA5KVDK9v
Tue Dec 27 08:49:55 2022 OpenSSL: error:1416F086:SSL routines:tls_process_server_certificate:certificate verify failed
Tue Dec 27 08:49:55 2022 TLS_ERROR: BIO read tls_read_plaintext error
Tue Dec 27 08:49:55 2022 TLS Error: TLS object -> incoming plaintext read error
Tue Dec 27 08:49:55 2022 TLS Error: TLS handshake failed
```

**Comment by Matej-Liska** [comment]
Hi have you found a solution?

**Comment by g-pane** [comment]
The author of the script don't mention anyware that the client certificate has 2 years validiy. NOWARE. There is NO SOLUTION, except to replace all the certificate BEFORE they expire, not AFTER. I have modify the script for make the vality to 10 years.. but was a damage for me, i have make another server and i have lost all the connection and i need to reaplce MANUALLY.
@angristan add someware a note for this or make the valitidy of the client certificate editable by the user
